src/pages/en/tutorial/3-components/2.md

```diff
@@ -163,38 +165,35 @@ import '../styles/global.css';
 
 <Box icon="question-mark">
 
-### Analyze the pattern
-
-1. What line of code do you need to write in an Astro component to **receive values** of `title`, `author`, and`date` **as props**?
-
-    || `const { title, author, date } = Astro.props;` ||
+### Test Yourself
+
+1. What line of code do you need to write in an Astro component's frontmatter to **receive values** of `title`, `author`, and`date` **as props**?
+
+  <MultipleChoice>
```

Comment:
We need two more spaces of indentation to nest this correctly under the question:
```suggestion
    <MultipleChoice>
```